src/Field/Integer.php

```diff
@@ -0,0 +1,43 @@
+<?php
+
+// vim:ts=4:sw=4:et:fdm=marker:fdl=0
+
+namespace atk4\data\Field;
+
+use atk4\data\Field;
+use atk4\data\ValidationException;
+
+/**
+ * Integer field type.
+ */
+class Integer extends Numeric
+{
+    /** @var string Field type for backward compatibility. */
+    public $type = 'integer';
+
+    /**
+     * Specify how many decimal numbers should be saved.
+     */
+    public $decimalNumbers = 0;
```

Comment:
we commonly use $decimal_number, no caps.. i think.

Reply:
addressed